### PR TITLE
Fix coding style in NBitcoin - Part 2

### PR DESCRIPTION
**Pull request by quantumagi**
This PR fixes the coding style in the following files:
- RPCClient.cs
- RPCClient.Wallet.cs
- pos_RPCClientTests.cs
- pos_RepositoryTests.cs

### CODE REVIEW

File: src/NBitcoin.Tests/pos_RPCClientTests.cs [hunk 1]
```diff
@@ -322,46 +314,48 @@ public void CanGetPeersInfo()
         [Trait("UnitTest", "UnitTest")]
         public void CanParseIpEndpoint()
         {
-            var endpoint = Utils.ParseIpEndpoint("google.com:94", 90);
+            System.Net.IPEndPoint endpoint = Utils.ParseIpEndpoint("google.com:94", 90);
```
Comment (fassadlr) [inline src/NBitcoin.Tests/pos_RPCClientTests.cs]: Just EndPoint here ;)

File: src/NBitcoin.Tests/pos_RepositoryTests.cs [hunk 2]
```diff
@@ -419,15 +406,19 @@ public void EnumerateRawStratisBlockcahinAndWriteResultsToFile()
             var pathFile = $@"{TestDataLocations.BlockFolderLocation}\compare-blocks.txt";
             File.WriteAllLines(pathFile, fileInserts);
         }
+        */
 
         private IEnumerable<string> ManuallyEnumerateTheBlockchainFile()
         {
             // read all bytes form the first block file
-            var byts = File.ReadAllBytes(TestDataLocations.Block0001Location);
+            byte[] byts = File.ReadAllBytes(TestDataLocations.Block0001Location);
```
Comment (fassadlr) [inline src/NBitcoin.Tests/pos_RepositoryTests.cs]: Fix spelling to bytes?

File: src/NBitcoin.Tests/pos_RepositoryTests.cs [hunk 3]
```diff
@@ -419,15 +406,19 @@ public void EnumerateRawStratisBlockcahinAndWriteResultsToFile()
...
+
             // the magic byte separator of blocks
-            var m = new byte[4] { 0x70, 0x35, 0x22, 0x05 };
+            byte[] m = new byte[4] { 0x70, 0x35, 0x22, 0x05 };
```
Comment (fassadlr) [inline src/NBitcoin.Tests/pos_RepositoryTests.cs]: Maybe just rename this magicBytes?

File: src/NBitcoin/RPC/RPCClient.Wallet.cs [hunk 4]
```diff
@@ -266,17 +239,17 @@ public Money GetBalance()
             };
         }
 
-        //NBitcoin internally put a bit in the version number to make difference between transaction without input and transaction with witness.
+        // NBitcoin internally put a bit in the version number to make difference between transaction without input and transaction with witness.
         private string ToHex(Transaction tx)
         {
             // if there is inputs, then it can't be confusing
-            if(tx.Inputs.Count > 0)
+            if (tx.Inputs.Count > 0)
```
Comment (fassadlr) [inline src/NBitcoin/RPC/RPCClient.Wallet.cs]: Do `.Any()` here

File: src/NBitcoin/RPC/RPCClient.Wallet.cs [hunk 5]
```diff
@@ -447,21 +417,20 @@ private IEnumerable<RPCAccount> AsRPCAccount(RPCResponse response)
             }
         }
 
-
         // listaddressgroupings
```
Comment (fassadlr) [inline src/NBitcoin/RPC/RPCClient.Wallet.cs]: Delete line

File: src/NBitcoin.Tests/pos_RepositoryTests.cs (hunk 2 above)
Comment (quantumagi) [reply src/NBitcoin.Tests/pos_RepositoryTests.cs]: Done.

File: src/NBitcoin.Tests/pos_RepositoryTests.cs (hunk 3 above)
Comment (quantumagi) [reply src/NBitcoin.Tests/pos_RepositoryTests.cs]: Done.

File: src/NBitcoin/RPC/RPCClient.Wallet.cs (hunk 4 above)
Comment (quantumagi) [reply src/NBitcoin/RPC/RPCClient.Wallet.cs]: Done.

File: src/NBitcoin/RPC/RPCClient.Wallet.cs (hunk 5 above)
Comment (quantumagi) [reply src/NBitcoin/RPC/RPCClient.Wallet.cs]: Done.

**Pull request merged**